### DISCUSSION TITLE
q-toggle svg icon strangness.

### DISCUSSION THREAD

**Discussion by tenwa** [quasarframework/quasar#12000] [General - Components / Directives / etc] [answered]
I've upgraded to the latest version, and my q-toggle icon seems to have been broken

```
  <q-toggle
    dense
    color="secondary"
    class="row q-pt-xs q-pb-none weapon-togle-font"
    :icon="iconName"
  />
```

the iconname returns

"img:" + require("assets/toggle-ranged.svg");

And it used to show the svg correctly, now it seems to be showing a very large svg, much bigger than toggle button size.

**Comment by MilosPaunovic** [comment]
You can temporarily solve the issue at hand by adding this class:

```css
.q-toggle__thumb img {
  height: 100%;
}
```

This will show the icon inside the toggle.

**Comment by rstoenescu** [comment]
Please open a ticket on this.
Thanks.

**Comment by rstoenescu** [comment, marked as answer]
Fix will be available in Quasar v2.4.8 and v1.17.3
